coffee-shop-service/src/index.js

```diff
@@ -57,6 +57,14 @@ async function getOrders() {
 
 }
 
+async function getAvailableBeans() {
+  let result = await client.query({
+    query: queries.GET_AVAILABLE_BEANS
+  });
+
+  return result.data.availableBeans;
+}
+
 function createOrder(customerId, item) {
```

Comment:
if we don't need these methods any more... lets delete / clean up.

If you agree... do this as a separate PR.